app/viewmodels/checkAnswers/sections/transportUnit/TransportUnitGiveMoreInformationSummary.scala

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package viewmodels.checkAnswers.sections.transportUnit
+
+import models.{CheckMode, UserAnswers}
+import pages.sections.transportUnit.TransportUnitGiveMoreInformationPage
+import play.api.i18n.Messages
+import play.api.mvc.Call
+import uk.gov.hmrc.govukfrontend.views.Aliases.{HtmlContent, Text}
+import uk.gov.hmrc.govukfrontend.views.viewmodels.content.Content
+import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.SummaryListRow
+import viewmodels.govuk.summarylist._
+import viewmodels.implicits._
+
+object TransportUnitGiveMoreInformationSummary {
+
+  def row(answers: UserAnswers, linkComponent: views.html.components.link)(implicit messages: Messages): Option[SummaryListRow] = {
```

Comment:
Could be worth making this a class and then Injecting the link component, it'll have to be added to the controller later on otherwise and will save having to pass it around everywhere